tests/input.js

```diff
@@ -11,26 +11,11 @@ let { x, y, ...z } = { x: 1, y: 2, a: 3, b: 4 };
 // Spread properties
 let n = { x, y, ...z };
 
-// do expressions
-let loop = do {
-  if(x > 10) {
-    'big';
-  } else {
-    'small';
-  }
-};
-
 // decorator function
 function dec(id){
   return (target, property, descriptor) => console.log('executed', id);
 }
 
-// class decorators and properties
-class Foobar {
-  @dec(1)
-  static foo = 1;
-}
-
```

Comment:
What was all of these ? Random tests?

Reply:
So we had these plugins to allow us to do certain things in JS that, as far as I can tell, we don't actually do in our projects. These tests are from the first commit of this repo and I don't know if they were actually added thoughtfully or if they were just copy&pasted from somewhere else, so I thought I'd cut the fat ✂️

Reply:
Before I commit I'll make doubly sure we don't use those plugins - I know they're not in Appearance-Extension or Phoenix-Client